ui.R

```diff
@@ -21,7 +21,7 @@ ui <- shinydashboardPlus::dashboardPage(
       dropdownBlock(
         id = "header_selection_dropdown",
         title = "Selection",
-        icon = icon("sliders"),
+        icon = icon("sliders-h"),
```

Comment:
@afwillia [here](https://github.com/Sage-Bionetworks/data_curator/pull/320) is the fontawesome related pr I was talking about. I think it's worthwhile to check if your current version of `fontawesome` will raise any warnings about the icon names. It's not a big deal, but using the compatible version and icon name can help to reduce the warnings.